scripts/stubs.py

```diff
@@ -4,5 +4,5 @@
 
 Definitions = Union[str, Tuple[str, ...]]
 Parts = Tuple[str, ...]
-Word = namedtuple("Word", "pronunciation, genre, etymology, definitions")
+Word = namedtuple("Word", "pronunciation, genre, etymology, definitions, variant")
```

Comment:
As there may be several variants, should we use plural here? If so, apply changes everywhere :)

Reply:
There might be several variant indeed ("mirent" -> mirer / mettre) but in the end we need to choose one to add the word in the HTML file... So not sure it's worth making it plural. It's a string.

Reply:
That's my point: variants could be a list of strings and we would have to handle "mirer" and "mettre". Maybe useless for now :)